app/controllers/admin/unattached_promo_registrations_controller.rb

```diff
@@ -32,16 +32,18 @@ def report
     referral_codes = params[:referral_codes]
     @reporting_interval = params[:reporting_interval]
     @event_types = params[:event_types]
+    @geo = params[:geo].present?
```

Comment:
There's probably a better name for this param here and also in the Promo::RegistrationStatsGeneratorService

Reply:
Preferably `@geo` is `@is_geo` to indicate a boolean type.